### DISCUSSION TITLE
Charge session management with SteVe API

### DISCUSSION THREAD

**Discussion by rsyrnicki** [steve-community/steve#1525] [General]
I managed to get the SteVe working as docker container to try it out and its really cool. Now I'm having a few questions:
- Are there more possible API calls than the ones listed in the /manager/v3/api-docs ? Like for changing charging profiles, starting / stoping remote transactions? If not I'm thinking about implementing some. 
- How can I access the current meter data? I know my wallboxes can deliver some data like Current.Import or phase info.
- Is there some kind of user manual explaining the UI? I'm somewhat familiar with the OCPP protocoll, still it woulc be a nice read to know what else I can do with it.

**Comment by juherr** [comment]
For api see #1000
